packages/tools/client-debugger/client-debugger-chrome-extension/src/react-components/AudienceView.tsx

```diff
@@ -25,12 +34,58 @@ export interface AudienceViewProps extends HasContainerId {
 export function AudienceView(props: AudienceViewProps): React.ReactElement {
 	const { containerId } = props;
 
-	// TODO: Post message requesting Audience summary
-	// TODO: Listen for Audience data updates
+	const messageRelay = React.useContext(MessageRelayContext);
+
+	if (messageRelay === undefined) {
+		throw new Error(
+			"MessageRelayContext was not defined. Parent component is responsible for ensuring this has been constructed.",
+		);
+	}
+
+	const [allAudienceMembers, setAllAudienceMembers] = React.useState<
+		Map<string, IClient> | undefined
+	>();
+	const [audienceHistory, setAudienceHistory] = React.useState<readonly unknown[]>();
```

Comment:
We should be able to get the correct type for the state in here, I think.

Reply:
I initially set as `const [audienceHistory, setAudienceHistory] = React.useState<readonly AudienceChangeLogEntry[]>();` (as `audienceHistory` from `FluidClientDebugger` is `readonly AudienceChangeLogEntry[];`) 

However, the linter complains that the directory is not reachable so for now I have set the type to `unknown`

Reply:
You might just need to reload your vscode and/or rebuild the 2 packages.

Reply:
💡🫡 Got it!

Reply:
Changed to `const [audienceHistory, setAudienceHistory] = React.useState<readonly AudienceChangeLogEntry[]>`.